docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -176,8 +176,8 @@ The Alert's `componentsProps` prop was deprecated in favor of `slotProps`:
 
 ```diff
  <Alert
--  componentsProps={{ closeButton: { testid: 'test-id' } }}
-+  slotProps={{ closeButton: { testid: 'test-id' } }}
+-  componentsProps={{ closeButton: { id: 'test-id' } }}
++  slotProps={{ closeButton: { id: 'test-id' } }}
```

Comment:
Probably better to use another id since it's not a test id anymore.

```suggestion
+  slotProps={{ closeButton: { id: 'close-id' } }}
```

Same as in the example below: https://github.com/mui/material-ui/pull/44636/files#diff-a5a0a00a518352d06d4ddc3013a1da70dcae1423ee73e024aae5649982f84e7eR1360

or just

```suggestion
+  slotProps={{ closeButton: { id: 'some-id' } }}
```